src/coreclr/hosts/corerun/CMakeLists.txt

```diff
@@ -32,4 +32,4 @@ else(CLR_CMAKE_HOST_WIN32)
     endif()
 endif(CLR_CMAKE_HOST_WIN32)
 
-install_clr(TARGETS corerun DESTINATIONS . COMPONENT runtime)
+install_clr(TARGETS corerun DESTINATIONS . COMPONENT hosts)
```

Comment:
Should we move these to the src/native/* directory structure since it is used by mono and coreclr? cc: @jkotas

Reply:
I think moving them to the `src/native` tree would be a good idea.

Reply:
Agree we should move them.  (This was discussed in https://github.com/dotnet/runtime/issues/43952).  Can we do it in a follow-up PR?

Reply:
Yeah, follow-up PR sounds good to me, but at least let's open an issue to keep track of that TODO work.

Reply:
Opened an issue https://github.com/dotnet/runtime/issues/62512

Reply:
It seems like I should tackle this follow up item before the CI follow up item? Or does it make a difference?

Reply:
@naricc I think they're pretty independent tasks.  Depends on if you love yaml or cmake more 😀